backends/cuda-ref/ceed-cuda-ref-operator.c

```diff
@@ -1374,13 +1374,14 @@ static int CeedSingleOperatorAssembleSetup_Cuda(CeedOperator op) {
   CeedOperatorField *input_fields;
   CeedOperatorField *output_fields;
   ierr = CeedOperatorGetFields(op, &num_input_fields, &input_fields,
-                               &num_output_fields, &output_fields); CeedChk(ierr);
+                               &num_output_fields, &output_fields); CeedChkBackend(ierr);
```

Comment:
`CeedChkBackend()` should be used instead of `CeedChk()`. `CeedChkBackend()` flags all errors as major/unrecoverable since we may be in an indeterminate or corrupted state.